charts/pulsar/templates/toolset-statefulset.yaml

```diff
@@ -126,7 +126,7 @@ spec:
       - name: proxy-ca
         secret:
           {{- if eq .Values.certs.internal_issuer.type "selfsigning" }}
-          secretName: "{{ .Release.Name }}-{{ .Values.tls.ca_suffix }}"
+          secretName: "{{ template "pulsar.certs.issuers.ca.secretName" . }}"
           {{- end }}
           {{- if eq .Values.certs.internal_issuer.type "ca" }}
           secretName: "{{ .Values.certs.issuers.ca.secretName }}"
```

Comment:
It looks like `secretName: "{{ template "pulsar.certs.issuers.ca.secretName" . }}"` should be used also here for consistency.